### PR TITLE
Group4 rpc datanode impl

### PR DESCRIPTION
**Pull request by efrederic**
Skeleton implementations of all Client -> DN RPC calls

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,12 @@ make
 in the test/ directory.
 A beginner's guide to using Google Test is located [here](https://github.com/google/googletest/blob/master/googletest/docs/Primer.md)
 
-To create an example interaction with the RPC server,
-run the namenode executable from build/rice-namenode.
-Then run something like `hdfs dfs -fs hdfs://localhost:port/ -mkdir foo`
-where port is the port used by the namenode (it will print the port used).
+Namenode:
+Run the namenode executable from build/rice-namenode. 
+Then run something like "hdfs dfs -fs hdfs://localhost:port/ -mkdir foo"
```
Comment (pelmers) [inline README.md]: I would put the command in backticks to make copy paste easier

File: README.md [hunk 2]
```diff
@@ -60,7 +60,12 @@ make
...
+where port is the port used by the namenode (it will print the port used)
+
+Datanode:
+Run the datanode executable from build/rice-datanode. 
+Then run something like "hdfs dfsadmin -shutdownDatanode hdfs://localhost:port/"
```
Comment (pelmers) [inline README.md]: same^^

**Review by pelmers — APPROVED** [review]
r+

**Comment by jfking** [comment]
Looks good. Merging.

**Pull request merged**